admin.py

```diff
@@ -374,6 +370,7 @@ class DonationForm(djforms.ModelForm):
   event = make_admin_ajax_field(tracker.models.Donation, 'event', 'event', initial=latest_event_id)
   class Meta:
     model = tracker.models.Donation
+    exclude = ('', '')
```

Comment:
Wait, so why do you have a two-tuple of empty strings for this? Isn't just an empty array/tuple sufficient?